### PR TITLE
Added a package for react-timer-mixin

### PR DESCRIPTION
**Pull request by ducky427**

### CODE REVIEW

File: react-timer-mixin/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# cljsjs/blockapps-js
```
Comment (Deraen) [inline react-timer-mixin/README.md]: This line refers another package.

Comment (ducky427) [inline react-timer-mixin/README.md]: Opps. I'll fix this.

Comment (ducky427) [inline react-timer-mixin/README.md]: I've fixed that and squashed that commit into 1.

**Comment by ducky427** [comment]
Thanks @Deraen!

**Pull request merged**